app/app/BaseUser.php

```diff
@@ -330,18 +329,13 @@ public static function getKeywords()
 		return '';
 	}
 
-	/**
-	* send verification email to user's email
-	*
-	*/
-	public static function sendVerificationEmail($user)
-	{
-		$body = "Final step...\r\n".
-				"Confirm your email address to complete your Accesslocator account. It's easy — just click on the link below or simply copy and paste it into your browser.\r\n".
-				config('app.url')."/signup/confirmEmail/".$user->email."/".$user->email_verification_token;
-		$subject = "Confirm your Accesslocator account, 
-		".$user->first_name." ".$user->last_name;
-		Emailer::send($user->email, $subject, $body);
+    /**
+	 * Generates the Confirmation Link for the New User.
```

Comment:
This generally looks good.  I like the switch from string literals spread around the code base to email templates in separate files because that seems better organized.  The HTML-based emails should look a lot better too.

It would be important to test that it doesn't get filtered as SPAM since that's our main problem even with plain text emails.  HTML-based emails are more likely to get filtered because HTML has more power to deceive its reader with hidden URL's for malicious links.  More detail at: https://www.quora.com/Are-HTML-emails-marked-as-spam-more-often-than-simple-text-ones

I just looked at this quickly but will look more closely tonight.

A few clear things to improve are:
- fix indentation.  It looks like your editor has 4 spaces = 1 tab when github is using 8 spaces = 1 tab.  Using tab characters instead of any regular spaces for indentation is ideal.

Low priority improvements:
- move inline styles from email templates to the head's style element


Just as an update from last night, I merged and tested a deployment using the gmail changes on app.accesslocator.com. I ran into some problems and ran out of time to resolve them before heading to work so app.accesslocator.com doesn't have any email changes right now.

Reply:
@joshi1983 Yea there is a problem with indentation and I have checked the all the three Emails and they are landing up directly in the inbox of the user. So, after deployment, you need to update the `.env` file with correct email settings. After adding env variables you need to run the following 
`php artisan clear-compiled`
`php artisan config:clear`

Restart the server and you should be good to go. Let me know if you still run into problem.